politeiawww/www.go

```diff
@@ -389,6 +389,12 @@ func _main() error {
 		return err
 	}
 
+	// Set up policy configs that might be setted in d.
```

Comment:
`setted` is not a word.

This call can't fail. www needs to come up despite this call failing or not. So by making this a policy type deal this opens quite a can of worms. Before cache, the inventory was a lazy call and required quite a bit of surgery. Let's open this for debate on matrix.

Reply:
Hey @marcopeereboom, now policy is lazy loaded within the policy call. Thanks for the explanation on the review!